### DISCUSSION TITLE
列表组件提供拖拽排序功能就好了

### DISCUSSION THREAD

**Discussion by 280206735** [tusen-ai/naive-ui#2746] [Ideas]
列表组件什么时候能提供拖拽排序功能啊？  
![image](https://user-images.githubusercontent.com/52560278/162132123-87cd27b2-5402-4414-96da-8985528b4a46.png)

**Comment by 07akioni** [comment]
目前没有加的计划，所以时间没有预期

**Comment by jahnli** [comment]
ref https://github.com/tusen-ai/naive-ui/issues/5069
